samples/snippets/create_analysis.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# [START contactcenterinsights_create_analysis]
+from google.cloud import contact_center_insights_v1
+
+
+def create_analysis(conversation_name: str) -> contact_center_insights_v1.Analysis:
+    # Construct an analysis.
+    analysis = contact_center_insights_v1.Analysis()
+
+    # Call the Insights client to create an analysis.
+    insights_client = contact_center_insights_v1.ContactCenterInsightsClient()
+    analysis_operation = insights_client.create_analysis(
+        parent=conversation_name, analysis=analysis
+    )
+    analysis = analysis_operation.result(timeout=600000)
```

Comment:
Same nit as in the other PR, the timeout seems a bit high to me.

Reply:
Oops great catch! I updated this timeout to 24 hours.